### ISSUE TITLE
HTML file script section preceeding spaces removed

### ISSUE THREAD

**Issue by ohxyz** [ony3000/prettier-plugin-brace-style#38]
## Dependency information

```
prettier-plugin-brace-style@0.6.2
prettier@3.2.5
```
.prettierrc
```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "singleQuote": true,
  "quoteProps": "preserve",
  "plugins": ["prettier-plugin-brace-style"]
}
```

## Steps to reproduce
Format following HTML file that has <script> section

```
<!doctype html>
<html lang="en">
  <head></head>
  <script type="module">
    import './first.js';
    import './seconde.js';
  </script>
  <body></body>
</html>
```
## The current behavior

Starting from the second `import`, preceeding spaces are removed. It becomes
```
<!doctype html>
<html lang="en">
  <head> </head>
  <script type="module">
    import './first.js';
import './seconde.js';
  </script>
  <body></body>
</html>
```
## The expected behavior

Expect it to be
```
<!doctype html>
<html lang="en">
  <head> </head>
  <script type="module">
    import './first.js';
    import './seconde.js';
  </script>
  <body></body>
</html>
```

**Comment by ony3000** [comment]
Hello! Thanks for reporting the issue!

HTML files are not yet supported in the current version, I plan to include HTML support in the next minor release. :pray:

**Comment by ony3000** [comment]
@ohxyz Fixed in `v0.7.0`, check it out!

**Comment by ohxyz** [comment]
> @ohxyz Fixed in `v0.7.0`, check it out!

Thanks. Worked great! 😉
